ui/Screen/Org.svelte

```diff
@@ -82,30 +82,21 @@
 
 <SidebarLayout
   dataCy="org-screen"
-  style="margin-top: calc(var(--topbar-height) + 33px)">
+  style="margin-top: calc(var(--topbar-height) + 200px)">
```

Comment:
Can we put this `200px` in a var? It's also used in `.banner`, so changing one of those will break the other..

Reply:
resolved in [80abf4a](https://github.com/radicle-dev/radicle-upstream/pull/494/commits/80abf4a62ab6607f079bd6c6c8d840b554742fae)